Views/backend/product_stream/view/selected_list/connect_windows.js

```diff
@@ -0,0 +1,29 @@
+//{block name="backend/product_stream/view/selected_list/window" append}
+Ext.define('Shopware.apps.ProductStream.view.selected_list.ConnectWindow', {
+    override: 'Shopware.apps.ProductStream.view.selected_list.Window',
+
+    createProductGrid: function() {
+        var me = this,
+            productGrid = this.callParent(arguments);
+
+        if (me.record.get('isConnect')) {
+            productGrid.searchField.disable();
+            productGrid.grid.columns.pop();
+        }
+
+        return productGrid
+    },
+
+    createSettingPanel: function() {
+        var me = this,
+            settingsPanel = this.callParent(arguments);
+
+        if (me.record.get('isConnect')) {
+            settingsPanel.nameField.disable();
```

Comment:
is it possible to foreach all fields in settingsPanel and then disable() or just set disabled for the whole settingsPanel?